card/lib/card/loader.rb

```diff
@@ -21,6 +21,40 @@ def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == "development"
+          update_machine_output_hack
+        end
+      end
+
+      def update_machine_output_hack
+        script = Card["*all+*script"]
+        if (mtime_output = script.machine_output_card.updated_at)
+          [ "wagn_mod.js.coffee", "wagn.js.coffee", "script_card_menu.js.coffee" ].each do |name|
+            mtime_file = File.mtime "#{Cardio.gem_root}/mod/03_machines/lib/javascript/#{name}"
+            if mtime_file > mtime_output
+              script.update_machine_output
+              return
+            end
+          end
+        end
+
+        style = Card["*all+*style"]
+        if (mtime_output = style.machine_output_card.updated_at)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.